specs/waku/v2/waku-filter.md

```diff
@@ -116,11 +126,14 @@ implementation, though a reasonable default is one minute.
 ### Next version
 
 - Fix: Ensure contentFilter and contentTopic are repeated fields, per implementation
+- Change: Add ability to unsubscribe from filters. Make `subscribe` an explicit boolean indication. Edit protobuf field order to be consistent with libp2p.
 
 ### 2.0.0-beta1
 
 Initial draft version. Released [2020-10-05](https://github.com/vacp2p/specs/commit/31857c7434fa17efc00e3cd648d90448797d107b)
 
+### 2.0.0-beta2
```

Comment:
Generally we put this first in reverse chronological order. I'd fill this in now, then once PR approved squash, use that commit it release hash, then do rebase+merge with two commits.

Reply:
Reversed order in 5a731c0. Will follow your suggested process to link to commit hash after PR approval.